src/ngMock/angular-mocks.js

```diff
@@ -2072,13 +2072,13 @@ function MockXhr() {
     var header = this.$$respHeaders[name];
     if (header) return header;
 
-    name = angular.lowercase(name);
+    name = angular.$$lowercase(name);
```

Comment:
Why keep the service, even as a private one? `String.prototype` methods should be enough.